modules/release-notes/con-relnotes-fixed-issues.adoc

```diff
@@ -220,6 +220,46 @@ With the release of the {product-short} 1.2.1 Helm chart, this is fixed.
 
 == Fixed security issues
 
+=== Fixed security issues in {product} 1.2.5
```

Comment:
@themr0c if instead of a "dump to console then copy-and-paste" approach we want to generate a single snip file for this, I'm happy to revise the PR to do that -- just need to know what filename you want here, and how to reference it in the co-relnotes-fixed-issues.adoc file.